### ISSUE TITLE
Add vi like navigation

### ISSUE THREAD

**Issue by daniel-vera-g** [pindexis/marker#64]
After doing `<Ctrl> + <Space>` you can navigate through the results with the up/down keys.  
It would be great to use the j/k keys. `fzf`for example uses `<Ctrl> + P` & `<Ctrl> + N` for navigation. This is very intuitive.

**Comment by weilbith** [comment]
Yes, it would be awesome to configure these key-bindings. So `<Ctrl> + j`/`<Ctrl> + k` are also quite popular. It's quite annoying to move the hands so far to the arrow keys. 🙊 
Not sure how [this](https://github.com/pindexis/marker/blob/443042cc0fcedaf09a9dc9274f45877ab462f8d5/marker/readchar.py#L20) is working here. 🤔

**Comment by bitsadmin** [comment]
Added support for Ctrl+J/K and submitted pull request to @pindexis.

https://github.com/pindexis/marker/pull/71
